sklearn/utils/estimator_checks.py

```diff
@@ -1471,24 +1516,30 @@ def check_estimators_unfitted(name, estimator_orig):
     # Common test for Regressors, Classifiers and Outlier detection estimators
     X, y = _boston_subset()
 
-    est = clone(estimator_orig)
+    estimator = clone(estimator_orig)
```

Comment:
we tend to avoid these cosmetic changes...

Reply:
ok... I thought it was ok when neighbouring code was modified...